test/e2e/testing-manifests/storage-csi/mock/csi-mock-driver-snapshotter.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: csi-mock
       containers:
         - name: csi-snapshotter
-          image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.2.1
```

Comment:
Should this be using the v5.0.0-rc1 image?

Reply:
I don't think so. The goal is to be as close as possible to the released csi-driver-host-path deployment..

The canary jobs cover the upcoming releases.